test/web-embed-lab/test-probes/dlo-rules/test.js

```diff
@@ -0,0 +1,59 @@
+/**
+DLORulesProbe tests that a well known array contains expected rule results
+*/
+class DLORulesProbe {
+  /**
+  @return {Object} data collected when the target embed script *is not* loaded
+  @return {Object.success} always true
+  */
+  async gatherBaselineData(){
+    console.log('DLO rules baseline')
```

Comment:
nit: semicolon missing.  I suspect our linter isn't working on these tests.

Reply:
Yep, the WEL isn't using the linter (or semicolons) so the new test is consistent with the other test probes from the WEL.